Bottom: c85c94a0181a926d576280bd29d50bcb8ced3055
Top:    f982be0c4c46b62333acb7cf6e028fc7a5ef7729
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-26 14:42:35 +0100

Refresh of hid-gtk-remove-prototype-for-n

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index 134d2fb..782b147 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -264,7 +264,6 @@ void ghid_get_pointer (gint *, gint *);
 void ghid_port_ranges_changed (void);
 gboolean ghid_port_ranges_pan (gdouble x, gdouble y, gboolean relative);
 void ghid_port_ranges_scale (void);
-void ghid_port_ranges_update_ranges (void);
 
 gboolean ghid_note_event_location (GdkEventButton * ev);
 gboolean have_crosshair_attachments (void);
